index.html

```diff
@@ -879,26 +878,26 @@ <h2 id="BaseAudioContext">
                 following the last sample-frame in the block of audio most
                 recently processed by the context's rendering graph. If the
                 context's rendering graph has not yet processed a block of
-                audio, then <a href=
-                "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>
-                has a value of zero.
+                audio, then <a data-link-for=
+                "BaseAudioContext"><code>currentTime</code></a> has a value of
+                zero.
               </p>
               <p>
-                In the time coordinate system of <a href=
-                "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>,
-                the value of zero corresponds to the first sample-frame in the
-                first block processed by the graph. Elapsed time in this system
-                corresponds to elapsed time in the audio stream generated by
-                the <a><code>BaseAudioContext</code></a>, which may not be
+                In the time coordinate system of <a data-link-for=
+                "BaseAudioContext"><code>currentTime</code></a>, the value of
```

Comment:
I think that `<code>` should be outside the a tag. Or might be possibly dropped completely?

But I see we do a lot of that elsewhere, so maybe ignore this.